js/index.js

```diff
@@ -62,6 +65,15 @@ function handlePostalCodeInput(codePostal, next) {
 	request.send()
 }
 
+/*
+contrat_de_travail <select> value can trigger the display
+of the heures_remunerees_volume <input> field.
+*/
+function handleTempsPartielSelect(contrat, next) {
+	document.querySelector('#temps_partiel_container').hidden = contrat === 'temps_plein'
```

Comment:
I think it's fine even for IE11 to use `.hidden =` with the css rule 
```css
[hidden] {
	display: none;
}
```
Or is it the direct property setter that is not supported ? setAttribute won't work, we'd need removeAttribute too.